cygnss_wetlands/cygnss/download.py

```diff
@@ -1,6 +1,7 @@
 import base64
 import datetime
 import json
+import os.path
```

Comment:
Should this import be simply `import os` ? Looks like both `os.path.getsize()` and `os.remove()` are called

(note: i just tested and it works as-is! I didn't realize this pattern was possible !)

Reply:
:o it is slightly concerning to me that this code works while only importing os.path lol